include/mbedtls/cmac.h

```diff
@@ -1,8 +1,10 @@
 /**
  * \file cmac.h
  *
- * \brief The Cipher-based Message Authentication Code (CMAC) Mode for
- *        Authentication.
+ * \brief This file contains CMAC definitions and functions.
```

Comment:
I'd rather have the full name and the abbreviation in the \brief block.

Reply:
The file brief description should refer to what the file contains. We provide the full name in the detailed description as a service and further clarification.